src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -68,12 +70,17 @@
 }
 
 .nav-search-results {
-  position: absolute;
```

Comment:
Removed as it was already overridden by the `position: fixed` below, so it wasn't actually doing anything.